libs/wingc/src/type_check.rs

```diff
@@ -57,15 +61,24 @@ pub enum SymbolKind {
 	Namespace(Namespace),
 }
 
+// Information about a variable in the environment
 #[derive(Debug, Clone)]
 pub struct VariableInfo {
+	// Type of the variable
 	pub _type: TypeRef,
+	// Can the variable be reassigned?
 	pub reassignable: bool,
+	// The phase in which this variable exists
+	pub flight: Phase,
 }
```

Comment:
nit: style (triple-slash is the convention for documentation comments in Rust)
```suggestion
/// Information about a variable in the environment
#[derive(Debug, Clone)]
pub struct VariableInfo {
	/// Type of the variable
	pub _type: TypeRef,
	/// Can the variable be reassigned?
	pub reassignable: bool,
	/// The phase in which this variable exists
	pub flight: Phase,
}
```